MultiVersions/Forge/main/vanillacookbook/lang/zh_tw.json

```
@@ -0,0 +1,162 @@
+	"item.vanillacookbook.cheesecake_slice": "乳酪蛋糕片",
+	"item.vanillacookbook.chocolate_cake_slice": "巧克力蛋糕片",
+	"item.vanillacookbook.chorus_cake_slice": "歌萊蛋糕片",
+	"item.vanillacookbook.ice_cream_cake_slice": "冰淇淋蛋糕片",
+	"item.vanillacookbook.mohnkuchen_slice": "罌粟籽蛋糕片",
+	"item.vanillacookbook.rose_cake_slice": "玫瑰蛋糕片",
+	"item.vanillacookbook.glow_berry_cake_slice": "螢光莓蛋糕片",
+	"item.vanillacookbook.rose_rock_candy": "玫瑰冰糖",
+	"item.vanillacookbook.polished_rose_rock_candy": "拋光玫瑰冰糖",
+	"item.vanillacookbook.mana_roll": "瑪納瑞士捲",
+	"item.vanillacookbook.tater_puffs": "炸薯球",
+	"block.vanillacookbook.berry_cake": "甜莓蛋糕",
+	"block.vanillacookbook.carrot_cake": "胡蘿蔔蛋糕",
+	"block.vanillacookbook.cheesecake": "乳酪蛋糕",
+	"block.vanillacookbook.chocolate_cake": "巧克力蛋糕",
+	"block.vanillacookbook.mohnkuchen": "罌粟籽蛋糕",
+	"block.vanillacookbook.ice_cream_cake": "冰淇淋蛋糕",
+	"block.vanillacookbook.rose_cake": "玫瑰蛋糕",
+	"block.vanillacookbook.chorus_cake": "歌萊蛋糕",
+	"block.vanillacookbook.book_cake": "食譜蛋糕",
+	"block.vanillacookbook.glow_berry_cake": "螢光莓蛋糕",
+	"block.vanillacookbook.pancake_stack": "鬆餅堆",
+	"block.vanillacookbook.brownie_tray": "方格布朗尼",
+	"vanillacookbook.tooltip.cheese": "解除所有狀態效果",
+	"vanillacookbook.tooltip.golden_fruit_salad": "治療 2 點生命值",
+	"vanillacookbook.tooltip.ice_cream": "冰涼止火",
+	"vanillacookbook.tooltip.chocolate_ice_cream": "冰涼止火",
+	"vanillacookbook.tooltip.berry_ice_cream": "冰涼止火",
+	"vanillacookbook.tooltip.neapolitan_ice_cream": "冰涼止火",
+	"vanillacookbook.tooltip.pumpsicle": "冰涼止火",
+	"vanillacookbook.tooltip.snow_horn": "冰涼止火",
+	"vanillacookbook.tooltip.potato_chips": "50% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.hot_potato_chips": "50% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.beetroot_chips": "50% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.apple_chips": "50% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.honey_candy": "30% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.berry_candy": "30% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.warped_gummy": "解除凋零效果",
+	"vanillacookbook.tooltip.chocolate": "30% 機率解除飢餓效果",
+	"vanillacookbook.tooltip.pitcher_praline": "解除挖掘疲勞效果",
+	"vanillacookbook.tooltip.stuffed_sea_pickle": "解除所有狀態效果",
+	"vanillacookbook.tooltip.milk_bottle": "解除所有狀態效果",
+	"vanillacookbook.tooltip.chocolate_milk": "解除所有狀態效果",
+	"vanillacookbook.tooltip.berry_milk": "解除所有狀態效果",
+	"vanillacookbook.tooltip.cheesecake_slice": "解除所有狀態效果",
+	"vanillacookbook.tooltip.rose_rock_candy": "解除劇毒效果",
+	"vanillacookbook.tooltip.polished_rose_rock_candy": "解除劇毒效果",
+	"vanillacookbook.tooltip.mana_roll": "解除所有狀態效果",
+	"vanillacookbook.tooltip.tater_puffs": "50% 機率治療 1 顆生命值",
+	"advancements.more_balanced_diet.title": "絕對均衡飲食",
+	"advancements.more_balanced_diet.description": "再次吃遍所有可以食用的東西！",
+	"advancements.cake.title": "智慧之食",
+	"advancements.cake.description": "合成一本食譜蛋糕，感謝你使用《Vanilla Cookbook》！",
+	"advancements.easter_egg.title": "復活節彩蛋！",
+	"advancements.easter_egg.description": "你找到了一顆復活節彩蛋...如假包換！",
+	"itemGroup.vanillacookbook": "Vanilla Cookbook"
```

只是創造模式的小強迫症，想看到 itemGroup 有翻譯（
不一定要採納
```suggestion
	"itemGroup.vanillacookbook": "原版烹飪食譜"
```